README.md

```diff
@@ -122,3 +122,33 @@ To clean up generate coverage information files, run
 	$> find . -name \*.profraw -type f -exec rm -f {} +
 
 Open `index.html` from `./target/debug/coverage/` folder to review coverage data. Since WASM build is not possible yet, SKIP_WASM_BUILD is required when running tests.
+
+## Runtime upgrades
+
+Substrate development framework supports forkless upgrades to the runtime. Every update should increment spec or impl fields of `RuntimeVersion` in `runtimeime/src/lib.rs`.
+
+### Build and optimize WASM runtime
+
+Use [srtool cli](https://github.com/chevdor/srtool-cli) to compile WASM runtime:
+
+	$> srtool build -r runtime/ --package da-runtime
+
+If needed, use `wasm-opt` from [Binaryen](https://github.com/WebAssembly/binaryen) to optimize size of a WASM runtime:
+
+	$> wasm-opt -Oz -o ./da_runtime.compact.wasm \
+		./runtime/target/srtool/release/wbuild/da-runtime/da_runtime.compact.wasm
+
+### Upgrade process
+
+Since we have block size and transaction weight limits, upgrade is a three step process:
+
+1. Use `sudo/sudoCall` to invoke `dataAvailability/submit_block_length_proposal` with increased block limits
```

Comment:
In docs, we should not recommend `sudo` calls. IMO, we should have a way to run upgrade using democracy pallet, which internally calls the above `sudo` calls. This will ensure that future upgrades can be done in a decentralized manner and can be done on canary/mainnet. 

We can keep these as part of devnet upgrades.